core/storage/base_model/gae_models.py

```diff
@@ -687,17 +711,15 @@ def has_reference_to_user_id(cls, user_id: str) -> bool:
         """
         return cls.query(cls.user_id == user_id).get(keys_only=True) is not None
 
-    # TODO(#13523): Change 'commit_cmds' to domain object/TypedDict to
-    # remove Any from type-annotation below.
     @classmethod
     def create(
             cls: Type[SELF_BASE_COMMIT_LOG_ENTRY_MODEL],
             entity_id: str,
             version: int,
             committer_id: str,
             commit_type: str,
-            commit_message: str,
-            commit_cmds: Union[Dict[str, Any], List[Dict[str, Any]], None],
+            commit_message: Optional[str],
```

Comment:
Slightly concerned about this. In what cases can the commit message be None? Perhaps we need to fix those.

Ditto below and in other files.

Reply:
Basically, when the collection is unpublished, commit_message can be accepted as None. 
https://github.com/oppia/oppia/blob/22e557ebd3d5394aacd9c30c45698fb30d8e5a6a/core/domain/collection_services.py#L956
Similarly in skill:
https://github.com/oppia/oppia/blob/22e557ebd3d5394aacd9c30c45698fb30d8e5a6a/core/domain/skill_services.py#L819
Similar case in exploration:
https://github.com/oppia/oppia/blob/22e557ebd3d5394aacd9c30c45698fb30d8e5a6a/core/domain/exp_services.py#L1422

Reply:
This is a create() method, isn't it? The examples you showed seem to be update methods, but I thought creation always has a commit message.

Reply:
Oh wait, sorry, this is creation of a version snapshot. OK, then it sounds like str/None is fine, thanks.